projects/netgrif-components-core/src/lib/side-menu/content-components/load-filter/abstract-load-filter.component.ts

```diff
@@ -21,7 +22,8 @@ export abstract class AbstractLoadFilterComponent extends AbstractCaseViewCompon
 
     protected constructor(@Inject(NAE_SIDE_MENU_CONTROL) protected _sideMenuControl: SideMenuControl,
                           protected _log: LoggerService,
-                          caseViewService: CaseViewService) {
+                          caseViewService: CaseViewService,
+                          protected _activatedRoute?: ActivatedRoute) {
```

Comment:
why is the activatedRoute attribute added if it is not used anywhere?

Reply:
Corrected.